src/back/auth/src/repo/table_session.cpp

```diff
@@ -89,15 +102,44 @@ void Session::UpdateTokens(const model::Session& s)
 }
 
 const storages::postgres::Query kMarkSessionInactive{
-	"UPDATE session SET active = false"
+	"UPDATE session SET active = false "
 	"WHERE id=$1",
 	storages::postgres::Query::Name{"mark_session_inactive"},
 };
 
+const storages::postgres::Query kMarkSessionInactiveAndCleanTokens{
+	"UPDATE session SET active=false, token='', accessToken='', refreshToken='', idToken='' "
+	"WHERE id=$1",
+	storages::postgres::Query::Name{"mark_session_inactive_and_clean_tokens"},
+};
+
 void Session::MarkInactive(const model::Session& s)
 {
 	_pg->Execute(
 		storages::postgres::ClusterHostType::kMaster, kMarkSessionInactive, s._id);
 }
 
+void Session::MarkInactiveAndCleanTokensById(const boost::uuids::uuid& sid)
+{
+	_pg->Execute(
+		storages::postgres::ClusterHostType::kMaster, kMarkSessionInactiveAndCleanTokens, sid);
+}
+
+void Session::Refresh(
+	const model::Session& data,
+	const boost::uuids::uuid& oldSessionId)
```

Comment:
Слово session есть в названии класса. Да и другого id с таким типом тут быть не может. Предлагаю укоротить.)
```suggestion
	const boost::uuids::uuid& oldId)
```

Reply:
Согласен:) Поправил.